ocaml/typing/typeclass.ml

```diff
@@ -1187,6 +1188,7 @@ and class_expr_aux cl_num val_env met_env virt self_scope scl =
       in
       class_expr cl_num val_env met_env virt self_scope sfun
   | Pcl_fun (l, None, spat, scl') ->
+      let l = transl_label l in
```

Comment:
Ditto here. This could be triggered with e.g.

```
class c ~(src_pos:[%src_pos]) = object end
```

You're still probably saved because the compiler will fail at `[%src_pos]`.

So maybe the best approach is just to comment the uses of `transl_label` as OK only because `[%src_pos]` causes a failure elsewhere. And add these tests to the testsuite. And leave a CR somewhere that we should fix up classes and `[%src_pos]`.